### ISSUE TITLE
[Accessibility] [core][fd-avatar-group] The popover should not be  truncated and the title should be multiple lines

### ISSUE THREAD

**Issue by Meggie886** [SAP/fundamental-ngx#13826]
### Describe your proposal

The popover in the avatar group should not be truncated. The title within the popover should support multiple lines of text, and the height of the title container should exceed that of the content.

<img width="755" height="863" alt="Image" src="https://github.com/user-attachments/assets/1c173592-a3b9-4bf0-b9f7-e86dae28c8af" />
<img width="467" height="686" alt="Image" src="https://github.com/user-attachments/assets/1923b938-b27b-4f13-bffe-8278664cca34" />

### Which versions of Angular and Fundamental Library for Angular are affected?

angular v20.        @fundamental-ngx/core: "0.58.0"

### Steps to Reproduce (for bugs)
1. open console in the browser 
2. change device
3. select a avatar with long title and then select the more button, choose one with long name.
<img width="755" height="863" alt="Image" src="https://github.com/user-attachments/assets/1c173592-a3b9-4bf0-b9f7-e86dae28c8af" />
<img width="467" height="686" alt="Image" src="https://github.com/user-attachments/assets/1923b938-b27b-4f13-bffe-8278664cca34" />

### StackBlitz example(s)

https://stackblitz.com/edit/es1dvirr?file=src%2Fapp%2Favatar-group-data-example.service.ts,src%2Fapp%2Fdefault%2Favatar-group-default-example.component.html,src%2Fapp%2Fdefault%2Fdefault%2Favatar-group-default-example.component.ts

### Declaration

- [x] I'm not disclosing any internal or sensitive information.

**Comment by InnaAtanasova** [comment]
This change depends on updates in Fundamental-styles first. The default behavior will remain truncation; however, we plan to introduce an input that allows the text to wrap across multiple lines when needed.

**Comment by InnaAtanasova** [comment]
The changes are done in https://github.com/SAP/fundamental-ngx/pull/14102

<img width="1757" height="1277" alt="Image" src="https://github.com/user-attachments/assets/64890b03-5bf8-420d-8fd0-bd02f3a630d5" />
<img width="1443" height="461" alt="Image" src="https://github.com/user-attachments/assets/ad473d62-f0fc-4b11-901b-6fc6591d00ae" />
<img width="828" height="702" alt="Image" src="https://github.com/user-attachments/assets/4b0779e6-9891-4190-8aa8-5a750b2fcb13" />
